pyquaticus/moos/pyquaticus_moos_bridge.py

```diff
@@ -403,6 +403,9 @@ def set_config(self, moos_config):
             self.flags.append(flag)
 
         self.scrimmage_pnts = np.asarray(self._moos_config.scrimmage_pnts, dtype=np.float32)
+        self.scrimmage = self.scrimmage_pnts[0][0]
```

Comment:
```suggestion
        # save the scrimmage X value as well as
        # the lower and upper point (used to determine distance to scrimmage line)
        self.scrimmage = self.scrimmage_pnts[0][0]
```

Reply:
Sorry I just realized that the scrimmage obs normalization will actually be affected. The max distance from the scrimmage line is set via `self.scrimmage` ([see here](https://github.com/mit-ll-trusted-autonomy/pyquaticus/blob/36518559327408b18190be32532f8753ac51c0d7/pyquaticus/envs/pyquaticus.py#L145)) and the current changes set self.scrimmage to the x-value of one of the points on the scrimmage line. This x-value will not be 80 meters (half the size of the longer dimension) because of the way the field coordinates are set. I will go fix this and then post a new commit to this pull request. I think I will do as you previously suggested and just set self.scrimage to `0.5*self.world_size[0]`.